tests/test_read_xml.py

```diff
@@ -14,7 +14,7 @@ class TestClassReadXML:
 
     def test_read(self):
         """tests read function of newspaper and HLNA Data"""
-        result = read_transkribus(DATA_PATH + "newspaper/test-annotation.xml")
+        result = read_transkribus(DATA_PATH + "newspaper/test-annotation.xml", True)
```

Comment:
It's probably a good idea to take a look at:
https://docs.python.org/3/library/pathlib.html#